docs-source/spring/content/development/cli/_index.md

```diff
@@ -560,3 +560,179 @@ The following is an example development workflow using the CLI:
+
+   2. Use the `compile-logs` command to get the logs that show the compilation progress:
+
+```cmd
+oractl:>help compile-logs
+NAME
+       compile-logs - Compilation progress
+
+SYNOPSIS
+       compile-logs --batch String --help 
+
+OPTIONS
+       --batch String
+       file ID returned from compile command. If not provided by default the last file
+       [Optional]
+
+       --help or -h 
+       help for compile-logs
+       [Optional]
+```
+
+   As previously mentioned, if the batch ID is not provided, then the logs of the most recent executed compilation are returned. For example:
+
+```cmd
+    oractl:>compile-logs
+
+    extracted: BOOT-INF/lib/spring-jcl-6.0.11.jar
+    extracted: BOOT-INF/lib/spring-boot-jarmode-layertools-3.1.2.jar
+    inflated: BOOT-INF/classpath.idx
+    inflated: BOOT-INF/layers.idx
+    ========================================================================================================================
+    GraalVM Native Image: Generating 'demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b.exec' (executable)...
+    ========================================================================================================================
+    For detailed information and explanations on the build output, visit:
+    https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
+    ------------------------------------------------------------------------------------------------------------------------
+```
+
+   If compile-logs returns a **Finished generating** message, download the **.exec** file. For example:
+
+```cmd
+    CPU:  Enable more CPU features with '-march=native' for improved performance.
+    QBM:  Use the quick build mode ('-Ob') to speed up builds during development.
+   ------------------------------------------------------------------------------------------------------------------------
+                       155.3s (8.2% of total time) in 169 GCs | Peak RSS: 5.34GB | CPU load: 0.70
+   ------------------------------------------------------------------------------------------------------------------------
+   Produced artifacts:
+   /uploads/demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b.temp/demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b.exec (executable)
+   ========================================================================================================================
+   Finished generating 'demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b.exec' in 31m 30s.
+   Compiling Complete.
+```
+
+   3. Use the `compile-download` command to download the **.exec** file generated. For example:
+
+```cmd
+oractl:>help compile-download
+    NAME
+       compile-download - Download executable file compiled
+
+    SYNOPSIS
+       compile-download --batch String --help 
+
+    OPTIONS
+       --batch String
+       file ID to download as executable file. If not provided by default the last file compiled
```

Comment:
Change to:

File ID to download as the executable file. If not provided by default, then the last file compiled.